lib/rack/twilio_webhook_authentication.rb

```diff
@@ -17,18 +17,20 @@ module Rack
   # doesn't validate then the middleware responds immediately with a 403 status.
 
   class TwilioWebhookAuthentication
-    def initialize(app, auth_token, *paths)
+    def initialize(app, auth_token, *paths, &block)
       @app = app
       @auth_token = auth_token
+      define_singleton_method(:getAuthToken, block) if block_given?
```

Comment:
:+1: Absolutely, be a Rubyist when writing Ruby code :)

Feedback incorporated in the commit just pushed. Thanks for the pointer.